### PR TITLE
[Ready for Review] Updating Issue Templates

### PR DESCRIPTION
**Pull request by sgibson91**
### Summary

This PR is updating the issue templates for the repository.

### List of changes proposed in this PR (pull-request)

* Added a config.yml file which will include links to the collaboration cafe documentation and gitter chat, alongside the new chapter and general issue templates
* Added a bug report template so it's clearer how people should report unexpected behaviour
* Updated (emojified) the general and new chapter templates

### What should a reviewer concentrate their feedback on?

- [x] Does anyone know how to test the emojis will render correctly or how to convert from `U+xxx` to `\Uxxxxxxxx`? @GeorgianaElena?
- [ ] Everything looks ok?

### Acknowledging contributors

- [x] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by GeorgianaElena** [comment]
@sgibson91 , I forked this repo and changed the default branch to be `update-issue-templates` and this is what it looks like when trying to open an issue:
turing-issue-templates

Live demo: https://github.com/GeorgianaElena/the-turing-way/issues/new/choose

**Comment by GeorgianaElena** [comment]
This page https://www.fileformat.info/info/unicode/char/1f4d6/index.htm shows different encodings for the open book unicode (U+1F4D6). Down the page there's a "Python source code" encoding of it (`"\U0001F4D6"` for the open book emoji).

I think if you use that it will render the emoji.

File: .github/ISSUE_TEMPLATE/CHAPTER_ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-name: New Chapter Template
+name: "U+1F4D6 New Chapter Template"
```
Comment (GeorgianaElena) [inline .github/ISSUE_TEMPLATE/CHAPTER_ISSUE_TEMPLATE.md:2]:
```suggestion
name: "\U0001F4D6 New Chapter Template"
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+contact_links:
+  - name: "U+2615 Collaboration Cafe"
```
Comment (GeorgianaElena) [inline .github/ISSUE_TEMPLATE/config.yml:2]:
```suggestion
  - name: "\u2615 Collaboration Cafe"
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+    url: https://github.com/alan-turing-institute/the-turing-way/blob/master/project_management/online-collaboration-cafe.md
+    about: Sign up for mentored sprints and co-working sessions with Turing Way community members
+  - name: "U+1F5E3 All other questions and general chat"
```
Comment (GeorgianaElena) [inline .github/ISSUE_TEMPLATE/config.yml:5]:
```suggestion
  - name: "\U0001F5E3 All other questions and general chat"
```

File: .github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
-name: General
-about: General issue
+name: "U+2728 General"
```
Comment (GeorgianaElena) [inline .github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.md:2]:
```suggestion
name: "\u2728 General"
```

**Comment by malvikasharan** [comment]
@all-contributors please add @GeorgianaElena for review. 🎉

**Comment by sgibson91** [comment]
Thank you for the review @malvikasharan! 💖 

Yes we do need it, that template is for creating content no opening issues. Though to should probably be moved to `book/templates/` to join the case study/notebook template.

**Pull request merged**